corehq/apps/ota/views.py

```diff
@@ -59,7 +59,7 @@ def get_restore_response(domain, couch_user, app_id=None, since=None, version='1
                             status=401)
 
     project = Domain.get_by_name(domain)
-    app = Application.get_db().get(app_id) if app_id else None
+    app = Application.get(app_id) if app_id else None
```

Comment:
Thanks @czue! Should I do the same [here][1], [here][2], [here][3] and [here][4]?

  [1]: https://github.com/dimagi/commcare-hq/blob/b9ee8fac2949ee7d0d84633d8367ae8d8479a6e7/corehq/apps/app_manager/dbaccessors.py#L67-L67
  [2]: https://github.com/dimagi/commcare-hq/blob/b9ee8fac2949ee7d0d84633d8367ae8d8479a6e7/corehq/apps/app_manager/dbaccessors.py#L93-L93
  [3]: https://github.com/dimagi/commcare-hq/blob/f1432609be4f48f794fbab2c50bdfe45b6c6e41b/corehq/apps/app_manager/tests/test_app_manager.py#L99-L99
  [4]: https://github.com/dimagi/commcare-hq/blob/fee1cb75ec600411c2cd806124e4c9b5c01ad586/corehq/apps/reports/filters/forms.py#L352-L352